cylc/flow/run_modes/skip.py

```diff
@@ -188,4 +188,4 @@ def skip_mode_validate(taskdefs: 'Dict[str, TaskDef]') -> None:
         message = 'The following tasks are set to run in skip mode:'
         for taskname in skip_mode_tasks:
             message += f'\n    * {taskname}'
-        LOG.warning(message)
+        LOG.info(message)
```

Comment:
We (Met Office) added this warning message in the skip mode proposal. The intention was to reduce the potential for users configuring tasks in skip mode (e.g. for testing), and forgetting to take them out of skip mode (e.g. for production).

I recently saw a workflow where dummy tasks were parameterised making this list rather long.

This warning was a bit dubious when we added it (there was some discussion at the time). On reflection, and with the new context of the warning triangles, I'm happy to demote it.

I think only Dave and I would be inclined to see this as controversial?

Reply:
Well it's not warning you anymore, and in your own words the fact that it was a warning was dubious to begin with.

Reply:
Sorry, I didn't notice this unresolved question before merging with two approvals.

I guess the point is *"Cylc will warn you about any tasks set to run in skip mode"* should now be changed to *"Cylc will flag any tasks..."* (or similar)?

Reply:
Well, not a biggie